todo_app/app.py

```diff
@@ -1,11 +1,43 @@
 from flask import Flask
-
+from flask.templating import render_template
 from todo_app.flask_config import Config
+from todo_app.data import session_items
+from flask import request
```

Comment:
Minor, but in case it's new note that you can chain imports from the same module with commas, i.e.
```python
from flask import Flask, request
```